Source/Renderer/AutomaticUniforms.js

```diff
@@ -1121,6 +1121,20 @@ define([
             }
         }),
 
+        /**
+         * The log of the current frustums far plane. Used for computing the log depth.
+         *
+         * @alias czm_logFarDistance
+         * @glslUniform
```

Comment:
I added the private tag instead in case we ever make this class public (likely for post-processing). It only has the one purpose and removes an extra log from the shaders.